sql/4_2_2-to-4_3_1_upgrade.sql

```diff
@@ -609,4 +609,15 @@ INSERT INTO `list_options` (`list_id`,`option_id`,`title`,`notes`,`seq`) VALUES
 
 #EndIf
 
-
+#IfNotRow2D list_options list_id lists option_id page_validation
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`) VALUES
+('lists', 'page_validation', 'Page Validation', 298, 1, 0, '', '', '', 0, 0, 1, '');
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`) VALUES
+('page_validation', 'add_edit_issue#theform', '/openemrValidation/openemr/interface/patient_file/summary/add_edit_issue.php', 0, 0, 0, '', '{form_title:{presence: true}}', '', 0, 0, 1, '');
```

Comment:
Hi, 
now it will be openemr/interface/patient_file/summary/add_edit_issue.php repreduced from $_PHP['self']